app/helpers/ems_cloud_helper/textual_summary.rb

```diff
@@ -53,7 +53,11 @@ def textual_instances
     num   = @ems.number_of(:vms)
     h     = {:label => label, :image => "vm", :value => num}
     if num > 0 && role_allows(:feature => "vm_show_list")
-      h[:link]  = url_for(:action => 'show', :id => @ems, :display => 'instances')
+      if controller.send(:restful?)
```

Comment:
Is this particular helper ever invoked from elsewhere? By my understanding, in here, we know exactly what the controller is, and we can jump straight to using `ems_cloud_path` -- no `restful?` check, and no `#{controller_name}`.